### ISSUE TITLE
Cant find cloned voice 

### ISSUE THREAD

**Issue by dillaryclump** [elevenlabs/elevenlabs-python#26]
I am getting this error:
 File "/home/dillon/PycharmProjects/aivideobot/main.py", line 36, in <module>
    generatevoice()
  File "/home/dillon/PycharmProjects/aivideobot/main.py", line 29, in generatevoice
    audio = generate(text=get_text_between_empty_lines("/home/dillon/PycharmProjects/aivideobot/script.txt"), voice="Jordan-Peterson", model="eleven_monolingual_v1")
  File "/home/dillon/PycharmProjects/aivideobot/venv/lib/python3.10/site-packages/elevenlabs/simple.py", line 113, in generate
    raise ValueError(f"Voice '{voice_str}' not found.")
ValueError: Voice 'Jordan-Peterson' not found.

when I try and run this code:
def generatevoice():
    audio = generate(text=get_text_between_empty_lines("/home/dillon/PycharmProjects/aivideobot/script.txt"), voice="Jordan-Peterson", model="eleven_monolingual_v1")
    play(audio)

if __name__ == '__main__':
    set_api_key("6c4822d20796a7560e05f7d1fd37a595")
    generatevoice()

**Comment by flavioschneider** [comment]
Hi @dillaryclump Do you see the voice when you do:
```py
from elevenlabs import voices, set_api_key

set_api_key('key')
print(voices())
```

**Comment by dillaryclump** [comment]
yes it does show all of the voices including the Jordan-Peterson one I want to select:
 Voice(voice_id='ELGRrJDKkD3CTEc9YcAU', name='Jordan-Peterson', category='cloned', description='', labels={}, samples=[VoiceSample(sample_id='gkRx0tICaiALJNrjTgDh', file_name='jordanpetersonvoice.mp3', mime_type='audio/mpeg', size_bytes=1570428, hash='d95b87f2b0836b394fc7ae973f874a6d')

is it that I am importing like this?

from elevenlabs import *

**Comment by dillaryclump** [comment]
ok Ive solved this. for some reason the name isnt working so I just did voice=voices()[-1] instead

**Comment by flavioschneider** [comment]
Alternatively, you can provide the `voice_id`. I wouldn't suggest calling `voices()` every time since that queries the API, either provide the id or pass the voice object

**Comment by isayahc** [comment]
> Hi @dillaryclump Do you see the voice when you do:
> 
> ```python
> from elevenlabs import voices, set_api_key
> 
> set_api_key('key')
> print(voices())
> ```

Thanks for this
